prettier.config.js

```diff
@@ -13,4 +13,20 @@
  * limitations under the License.
  */
 
-module.exports = require('@sumup/foundry/prettier')();
+module.exports = require('@sumup/foundry/prettier')(
+  {},
+  {
+    quoteProps: 'consistent',
+    proseWrap: 'always',
+    overrides: [
+      {
+        files: '*.md',
+        options: { parser: 'markdown' },
+      },
```

Comment:
I thought prettier inferred the parser automatically based on the file extension 🤔